src/int128.h

```diff
@@ -0,0 +1,60 @@
+#ifndef SECP256K1_INT128_H
+#define SECP256K1_INT128_H
+
+#if defined HAVE_CONFIG_H
+#include "libsecp256k1-config.h"
+#endif
+
+#include "util.h"
+
+#if defined(SECP256K1_INT128_NATIVE)
+#include "int128_native.h"
+#elif defined(SECP256K1_INT128_STRUCT)
+#include "int128_struct.h"
+#else
+#error "Please select int128 implementation"
+#endif
+
+static SECP256K1_INLINE uint64_t secp256k1_umulh(uint64_t a, uint64_t b);
+
+static SECP256K1_INLINE int64_t secp256k1_mulh(int64_t a, int64_t b);
```

Comment:
This function does not appear to be used outside of `src/int128_struct_impl.h`, perhaps it can be considered a private implementation detail there, rather than exposing it?

Reply:
Done.